browser/ui/views/speedreader/speedreader_bubble_util.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_SPEEDREADER_SPEEDREADER_BUBBLE_UTIL_H_
+#define BRAVE_BROWSER_UI_VIEWS_SPEEDREADER_SPEEDREADER_BUBBLE_UTIL_H_
+
+#include <memory>
+
+#include "ui/gfx/font.h"
+#include "ui/views/controls/link.h"
+
+namespace gfx {
+class FontList;
+}  // namespace gfx
+
+namespace views {
+class StyledLabel;
+}  // namespace views
+
+namespace speedreader {
+
+// Line height for multiline labels.
+extern const int kLineHeight;
```

Comment:
you can define these right there `constexpr int kLineHeight = 16;`

Reply:
fixed